src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -482,7 +483,7 @@ private static VscConverterStationMapData toMapData(VscConverterStation vscConve
     private static LoadMapData toMapData(Load load) {
         Terminal terminal = load.getTerminal();
         LoadMapData.LoadMapDataBuilder builder = LoadMapData.builder()
-            .name(load.getNameOrId())
+            .name(load.getOptionalName().orElse(StringUtils.EMPTY))
```

Comment:
orElse(null) ?

This should be done for all equipments type, and not only for the load ...

Reply:
done